src/main/java/com/aws/greengrass/clientdevices/auth/infra/NetworkState.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.infra;
+
+import com.aws.greengrass.mqttclient.CallbackEventManager;
+import com.aws.greengrass.mqttclient.MqttClient;
+import software.amazon.awssdk.crt.mqtt.MqttClientConnectionEvents;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+import javax.inject.Inject;
+
+public final class NetworkState {
+    public static final NetworkConnectivityState NETWORK_UP = NetworkConnectivityState.NETWORK_UP;
+    public static final NetworkConnectivityState NETWORK_DOWN = NetworkConnectivityState.NETWORK_DOWN;
```

Comment:
This variables seem a bit redundant maybe we can just reference the enum direclty

Reply:
I just did it as a shorthand. But that's fine. Removed and shorted `NetworkConnectivityState` to just `ConnectivityState`